config/zones.json

```diff
@@ -4112,6 +4112,30 @@
     },
     "timezone": "America/Lima"
   },
+  "PF": {
+    "bounding_box": [
+      [
+        -155.0369766919999,
+        -28.14120859199994
+      ],
+      [
+        -134.44298255099994,
+        -7.450127862999921
+      ]
+    ],
+    "capacity": {
+      "hydro": 49.4,
```

Comment:
48.2 MW in 2019 in https://www.service-public.pf/sde/wp-content/uploads/sites/15/2021/02/BEPF-2019-VF_online_mail_light_compressed.pdf page 17

Reply:
48.2 MW is only the capacity on Tahiti.

There's also 1.2 MW installed in the Marquesas Islands (on Hiva Oa, Nuku Hiva and Fatu Iva/Fatu Hiva) and 0.06 MW on Ra’iātea/Raiatea -- see p. 27, and also the map p. 26 ("Puissance hydraulique : 48.3 MW").